src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('span.population')];
+// const onePopulation = document.queryElementById
```

Comment:
This line is commented out and seems to contain a typo in the method name. It should be `document.getElementById` instead of `document.queryElementById` if you intend to use it.